vendor/k8s.io/kubernetes/pkg/api/v1/types.generated.go

```diff
@@ -16090,16 +16090,19 @@ func (x *ISCSIVolumeSource) CodecEncodeSelf(e *codec1978.Encoder) {
 		} else {
 			yysep2 := !z.EncBinary()
 			yy2arr2 := z.EncBasicHandle().StructToArray
-			var yyq2 [7]bool
+			var yyq2 [10]bool
```

Comment:
https://github.com/openshift/origin/pull/14112 picked in API changes without regenerating files.